src/templates/layout.hbs

```diff
@@ -51,8 +51,12 @@
             {{> "content/cardmenu/cardmenu" menuitem=subPages}}
           {{/if}}
         </div>
-
         {{> "content/prev_next/prev_next" prev=previousPage next=nextPage}}
+        {{#if changed}}
```

Comment:
I like the idea of having a general part that holds meta informations. But then we have to toggle the display in `src/components/content/meta-info/meta-info.hbs`.